app/models/solidus_paypal_braintree/response.rb

```diff
@@ -18,18 +19,14 @@ def build(result)
       def build_success(result)
         transaction = result.transaction
 
-        test = true
-        authorization = transaction.id
-        fraud_review = nil
-        avs_result = nil
-        cvv_result = nil
-
         options = {
-          test: test,
-          authorization: authorization,
-          fraud_review: fraud_review,
-          avs_result: avs_result,
-          cvv_result: cvv_result
+          test: !Rails.env.production?,
```

Comment:
Unfortunately we don't have access to anything from either Braintree nor from Solidus to set this value correctly. ActiveMerchant sets this value to `false` per default. As nobody consuming this response actually uses this value I think we can just remove it and rely on the ActiveMerchant default behavior here.